### PR TITLE
Fixes 9 24 24 v2

### PR DESCRIPTION
**Pull request by willkessler**
- new graphs on dashboard
- new AI help panel
- Attempts to fix clipping issues
- many small cosmetic fixes
- some refactors
- added new category filters on incidents page

### CODE REVIEW

File: src/lib/components/ActionItems.svelte
Comment (amkoo7) [inline src/lib/components/ActionItems.svelte:1]: Imo the text-grey-500 is a bit too strong/bold for the Vehicles table (and for tables generally).
Also, this doesn't match the font styling for the Federal and State filings which are text-gray-900. Between the two, the 500 is better.

File: src/routes/manage/incidents/+page.svelte [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
- import { incidents, getIncidentById, getIncidentTitle, getTypeColor, getStatusColor, getPriorityColor } from '$lib/data/incidentData';
+ import { incidents, getIncidentById, getIncidentsByCategory, 
+        getIncidentTitle, getTypeColor, getStatusColor, getPriorityColor } from '$lib/data/incidentData';
  import { drivers, getDriverById } from '$lib/data/driverData';
  import { vehicles, getVehicleById, getVehicleDriver } from '$lib/data/vehicleData';
 
- let activeTab = 'all events';
- const tabs = ['All events', 'Needs attention', 'Upcoming'];
+ const categories = ['On the Road', 'Maintenance', 'Records', 'All Incident Categories', ];
+ let activeCategory = 'On the Road';
```
Comment (amkoo7) [inline src/routes/manage/incidents/+page.svelte:13]: @willkessler should the active button be "All Incidents"? Currently when you go to the incidents manager none of the tabs are selected but it shows all incidents.

**Review by amkoo7 — APPROVED** [review]
@willkessler left a few comments. No bugs so will approve but some may result in additional work.

File: src/lib/components/ActionItems.svelte
Comment (willkessler) [reply src/lib/components/ActionItems.svelte:1]: i don't understand which lines you're referring to. if it's in the header than text-gray-400 is so close to its bg you can't really read it. and if if's the action button we make the text lighter on it appears to be disabled.

File: src/routes/manage/incidents/+page.svelte (hunk 1 above)
Comment (willkessler) [reply src/routes/manage/incidents/+page.svelte:13]: i think this PR is missing the code fixes that select On the Road as the active button first.

**Pull request merged**